lib/executables/exec.js

```diff
@@ -1,24 +1,63 @@
+/* eslint-disable no-use-before-define */
+const path = require('path');
 const { print } = require('../helpers/theme.js');
 const utils = require('../helpers/utils.js');
 const prompts = require('../helpers/prompts.js');
 const { Emitter } = require('../helpers/emitter.js');
 
 exports.runExec = async function runExec(componentPath, functionName, fixtureKey, action) {
-  const path = await prompts.getActionPath(componentPath, action);
-  const fn = await prompts.setupFunction(functionName, path);
+  const verifyOrAction = functionName || (action ? 'action' : await prompts.verifyOrAction());
   const fixture = await prompts.setupFixture(fixtureKey, componentPath);
+  const emitter = new Emitter();
   const cfg = fixture.cfg || {};
   const msg = fixture.msg || {};
   const snapshot = fixture.snapshot || {};
-  const emitter = new Emitter();
-
   utils.provideProcessWithEnvVars();
+  let fn;
+
+  if (verifyOrAction === 'verifyCredentials' || verifyOrAction === 'verify') {
+    // eslint-disable-next-line import/no-dynamic-require, global-require
+    fn = require(path.resolve(componentPath, 'verifyCredentials'));
+    if (Object.keys(fn).length) fn = fn.verify;
```

Comment:
`fn.verify` might not exist/be exported properly

```suggestion
    if ('verify' in fn) fn = fn.verify;
```

Maybe throw an error if it can't be found